evalml/tests/component_tests/test_time_series_imputer.py

```diff
@@ -658,3 +658,20 @@ def test_imputer_nullable_handling_noop_for_non_impute_methods(
         assert y_d.ww.schema == original_y_schema
     else:
         assert y_d.ww.schema != original_y_schema
+
+
+@pytest.mark.parametrize(
+    "nullable_ltype",
+    ["BooleanNullable", "IntegerNullable", "AgeNullable"],
+)
+@pytest.mark.xfail(strict=True, raises=ValueError)
+def test_time_series_imputer_nullable_type_incompatibility(
```

Comment:
@chukarsten @eccabay @jeremyliweishih I'm still adding the rest of the xfail tests but wanted to run this by yall. 

- I have the xfail set to only expect `ValueError`, but unfortunately, we can't specify the expected message. Would people prefer I get as specific about what the expected message is? 
- The `strict` parameter means that our test suite will fail when tests start passing - otherwise they just show as `XPASS` which I think would be easy to miss. But if we want to avoid blocking the update checker, we could set this to False and just periodically remember to check. I think that's probably a halfway point between what I'm suggesting and the fully manual checking process.

Reply:
This method of xfail seems like the best to me!

I'm a fan of keeping `strict=True`. It's really easy to miss an `XPASS`, and we don't have any way of flagging them at this point. I'm pretty sure we already have a bunch of `XPASS`ing tests that we might want to revisit...